charts/argo-rollouts/templates/dashboard/clusterrole.yaml

```diff
@@ -43,6 +43,7 @@ rules:
   - apiGroups:
       - apps
     resources:
+      - deployments
```

Comment:
Looks good to me. How about making this consistent to the cluster role as in the argo-rollouts repo:

https://github.com/argoproj/argo-rollouts/blob/55a041a6b3d088cf9ddcfc4121ad8c97880489db/manifests/role/argo-rollouts-clusterrole.yaml#L60-L69

Reply:
Sure will do!